chaoscraft-mod/src/main/kotlin/dev/strrl/chaoscraft/mod/show/UsageRender.kt

```diff
@@ -4,10 +4,13 @@ class UsageRender(
     private val resourceName: String,
     private val usagePercentage: Double,
 ) {
+    private val length = 20
     fun render(): String {
-
-        val used = "||||"
-        val unused = "     "
-        return "$resourceName: [$used$unused] 10%"
+        val usedLength = (length * usagePercentage).toInt()
```

Comment:
Please resolve `usagePercentage > 1.0` and `usagePercentage < 0`, which might break this program right now.
 
For `usagePercentage > 1.0`, just treat it as `1.0` when rendering the progress bar, but still show the exact number like `130%`.

For `usagePercentage<0`, just treat it as `0.0` at all of the places.